src/main/java/frc/robot/commands/drive/DoubleStationAlignCommand.java

```diff
@@ -58,7 +62,11 @@ public void initialize() {
 
         yController.setSetpoint(targetY);
         thetaController.setSetpoint(targetTheta.getRadians());
-        elevatorSubsystem.setPosition(ElevatorConstants.ElevatorPositions.DOUBLE_STATION);
+        if(ledSubsystem.getPieceTarget() == LEDSubsystem.PieceTarget.CUBE){
```

Comment:
you could make this niftier with a ternary: 


`elevatorSubsystem.setPosition(ledSubsystem.getPieceTarget() == LEDSubsystem.PieceTarget.CUBE ? ElevatorConstants.ElevatorPositions.DOUBLE_STATION_CUBE : ElevatorConstants.ElevatorPositions.DOUBLE_STATION_CONE);`